templates/frontend/objects/monograph_full.tpl

```diff
@@ -390,13 +390,13 @@
 						{assign var=publicationDates value=$publicationFormat->getPublicationDates()}
 						{assign var=publicationDates value=$publicationDates->toArray()}
 						{assign var=hasPubId value=false}
```

Comment:
I don't know about hasPubId, but I think that all publication format identifiers are listed in the right sidebar and the book doi is not listed/considered yet, but should be displayed below the title + authors at the top of the book page.